_methods/participatory-design.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: card
+title: Participatory design
+permalink: /fundamentals/participatory-design/
+redirect_from:
+  - /participatory-design/
+description: Design processes that invite participation are more sustainable in the long run.
+category: Fundamentals
+what: An approach to design emphasizing shared ownership, education, and collaboration.
+why: To ensure more just, equitable, and sustainable design processes. To encourage the creation of products and services reflecting the preferences of the people who must use them.
```

Comment:
What I understand to be "participatory design" takes a much stronger stance than "reflecting the preferences." The Scandinavian forms, for example, insist that the people who must use the tools are included at every step of the design process.

Reply:
@esgoodman are you suggesting we update this, or would it be enough to provide a link/reference that describes the Scandinavian forms? My goal is to describe how 18F practices this!

Reply:
I'd include that thing you just said about there being a range of participatory practices and that we occupy a specific point on that range.

Reply:
Collaborative design...where the collaboration is between different groups not just different people.

Reply:
Let's go with **collaborative design**